Dockerfile

```diff
@@ -16,4 +16,3 @@ COPY --from=builder /app/k6 /bin/
 COPY /scripts /scripts/
 COPY /proto /proto/
 COPY /locations /locations/
-COPY /precomputed /precomputed/
```

Comment:
Should we bake the precomputed files into the image by default? I guess the idea is that the user enters their own locations, and having these files baked in might not make it immediately clear if the volume was not set up correctly, so I removed it.